scipts/query_db.py

```diff
@@ -75,8 +75,9 @@ def main(args):
 
     for query in sorted(queries, key=itemgetter(7)):
         vcf_entry = query[8].rstrip()
-        sys.stdout.write("{};OCC={};FRQ={}\n".format(vcf_entry, query[7],(query[7]/float(len(dataBases))) ) )
-
+        content=vcf_entry.split("\t")
+        content[7]="{};OCC={};FRQ={}".format(content[7], query[7],(query[7]/float(len(dataBases) ) ))
```

Comment:
can you be a a bit more explicit here, and save in variables with clear name `content[7]` and `query[7]`? This will help a lot make the code easily to understand/maintain